docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  jetfish-pg:
+    image: postgres:latest
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
Keep postgres called postgres as a container, calling it `jetfish-pg` is pointless jargon. If you want a specific container name, use the `container_name` property.

Also this seems like a very short postgres section. A healthcheck for databases is recommended so dependent containers do not start before the backing DB is up and healthy.